src/utils/_variables.scss

```diff
@@ -0,0 +1,16 @@
+$cell-size: 100px;
+$container-padding: 10px;
+$cell-gap:1px;
+
+$cell-color: #eee;
+$cell-border-color: #000;
+$hover-color: #ffc0cb;
+
+$weekdays:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
```

Comment:
For some reason, I haven't thought of moving $weekdays variable into variables file. But it makes perfect sense!!